Jenkinsfile

```diff
@@ -73,10 +28,19 @@ spec:
     timeout(time: 90, unit: 'MINUTES')
   }
 
+  tools {
+     maven "apache-maven-latest"
```

Comment:
am not sure if i like this as it will "auto-update" once somebody at eclipse does random decisions